src/lib/device/physical_dev.cpp

```diff
@@ -360,6 +362,8 @@ void PhysicalDev::load_chunks(std::function< bool(cshared< Chunk >&) >&& chunk_f
             cinfo->checksum = info_crc;
 
             auto chunk = std::make_shared< Chunk >(this, *cinfo, cslot);
+            m_chunk_data_area.insert(
+                ChunkInterval::right_open(cinfo->chunk_start_offset, cinfo->chunk_start_offset + cinfo->chunk_size));
```

Comment:
So without this upon restart of homestore, if we add new chunk, it can overlap with each other?

Reply:
Yes without this , new chunks will overlap start offset.

Reply:
so we dont see this issue in HO repl_dev just because the chunk-info(offset) is not used there ? as we dont really care the location order between chunks?